Sources/ProjectDescription/TestAction.swift

```diff
@@ -35,6 +35,9 @@ public struct TestAction: Equatable, Codable {
     /// List of diagnostics options to set to the action.
     public var diagnosticsOptions: [SchemeDiagnosticsOption]
 
+    /// List of testIdentifiers to skip to the test
+    public var skippedTestIdentifiers: [String]?
```

Comment:
What about renaming this one to `skippedTests` to align with the property in `XcodeProj`?

Reply:
Thank you. I applied the suggestion. df33376